### ISSUE TITLE
1.4.0. beta4 - Connectivity bar not visible on high DPI settings

### ISSUE THREAD

**Issue by din3** [aliascash/alias-wallet-archive#31]
Known issue. Can be fixed with a workaround but would be great if the issue could go.
 
![1](https://user-images.githubusercontent.com/38725493/40996125-b89a05a0-6909-11e8-89d2-0f96e25e93ed.png)
![2](https://user-images.githubusercontent.com/38725493/40996126-b8bb3a18-6909-11e8-8882-a8e1a1697ba1.png)

**Comment by shbli** [comment]
@spectrecoin let me know if this should be fixed for 1.4.0 then tag it. Otherwise tag it with 1.4.1 or some other release.

**Comment by shbli** [comment]
@din3 what workaround are you applying?

It seems to me you are running the wallet on a server. it may be better then to use the command line wallet instead of the GUI one. the Command line wallet is faster.

**Comment by din3** [comment]
Workaround is as, creating a cmd file with the following input:

@echo off
set QT_AUTO_SCREEN_SCALE_FACTOR=0
start spectre.exe

I used a VM to artificially cause the bug to make screenshots.
But I had to help 3-4 people out in Discord in the past weeks which havent been able to see the bar.
And making them understand how to use the GUI is already time consuming. Doing it with a CLI is just not in there for newcomers.
